index.php

```diff
@@ -44,66 +44,25 @@ function makeCoffee($type = "cappuccino") {
   echo makeCoffee(null);    // Making a cup of .
   echo makeCoffee("mocha"); // Making a cup of mocha.
   ?>
-
-  <br/>
-
+  
   <?php

```

Comment:
Nice one :) You could also extract the php script in a separate file if you want like here: https://github.com/dftopalova/WEB2020-php/pull/1